src/components/dashboard/ReceiveToAddress.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+import React from 'react'
+import { Image, View } from 'react-native'
+import GoodWallet from '../../lib/wallet/GoodWallet'
+import InputText from '../common/form/InputText'
+import { Section, Text, Wrapper } from '../common'
+import TopBar from '../common/view/TopBar'
+import { withStyles } from '../../lib/styles'
+import { getDesignRelativeHeight } from '../../lib/utils/sizes'
+import normalize from '../../lib/utils/normalizeText'
+import illustration from '../../assets/Signup/maginLinkIllustration.svg'
+import CopyButton from '../common/buttons/CopyButton'
+
+export type TypeProps = {
+  screenProps: any,
+  navigation: any,
+  styles: any,
+}
+
+const ReceiveToAddress = (props: TypeProps) => {
+  const { screenProps, styles } = props
```

Comment:
1. pls decompose props in function declaration
2. GoodWallet.account reference is used twice.
pls decompose to constant before rendering

e.g. 
```
const { account } = GoodWallet

return (
  // ...
  <CopyButton style={styles.confirmButton} toCopy={account} onPressDone={screenProps.goToRoot} />
  // ...
```

Reply:
done